Makefile

```diff
@@ -103,15 +103,15 @@ for-all:
 check-uniform-dependencies:
 	./ci/check_uniform_dependencies.sh
 
-OT_CORE_VERSION := $(shell grep "otelcol_version: .*" otelcolbuilder/.otelcol-builder.yaml | cut -f 4 -d " ")
+OT_CORE_VERSION := $(shell grep "version: .*" otelcolbuilder/.otelcol-builder.yaml | cut -f 4 -d " ")
 OT_CONTRIB_VERSION := $(shell grep --max-count=1 '^  - gomod: github\.com/open-telemetry/opentelemetry-collector-contrib/' otelcolbuilder/.otelcol-builder.yaml | cut -d " " -f 6 | $(SED) "s/v//")
 # usage: make update-ot OT_CORE_NEW=x.x.x OT_CONTRIB_NEW=y.y.y
 .PHONY: update-ot
 update-ot: install-gsed
 	@test $(OT_CORE_NEW)    || (echo "usage: make update-ot OT_CORE_NEW=x.x.x OT_CONTRIB_NEW=y.y.y"; exit 1);
 	@test $(OT_CONTRIB_NEW) || (echo "usage: make update-ot OT_CORE_NEW=x.x.x OT_CONTRIB_NEW=y.y.y"; exit 1);
 	@echo "Updating OT core from $(OT_CORE_VERSION) to $(OT_CORE_NEW) and OT contrib from $(OT_CONTRIB_VERSION) to $(OT_CONTRIB_NEW)"
-	$(SED) -i "s/\(otelcol_version:\) $(OT_CORE_VERSION)$$/\1 $(OT_CORE_NEW)/" otelcolbuilder/.otelcol-builder.yaml
+	$(SED) -i "s/\(version:\) $(OT_CORE_VERSION)$$/\1 $(OT_CORE_NEW)/" otelcolbuilder/.otelcol-builder.yaml
```

Comment:
As discussed, let’s not use sed for editing yaml files 😅

Reply:
yeah, it was `sed` originally, I changed to use `yq` in some places.

Reply:
@echlebek I used yq to edit the version in the builder config. Replacing the version in all the dependencies listed in the builder config using yq seems tricky. Open to suggestions. Posting this comment here, because can't seem to comment in the previous thread for some reason.

Reply:
I was able replace sed with yq for the builder config here - https://github.com/SumoLogic/sumologic-otel-collector/pull/1714/commits/fa95a954dd7e6f4c434d3f6d4d2bcfec7639db7f